src/components/Navigation/Ui/ComponentButtons.js

```diff
@@ -65,13 +64,10 @@ export default class ComponentButtons extends Component {
     const scores = Definition.computeScores(definition)
     const isDefinitionEmpty = Definition.isDefinitionEmpty(definition)
     const isSourceEmpty = Definition.isSourceEmpty(definition)
-    const isCurated = Definition.isCurated(definition)
-    const hasPendingCurations = Definition.hasPendingCurations(definition)
+
     return (
       <div className="list-activity-area">
         {scores && <ScoreRenderer scores={scores} definition={definition} />}
-        {isCurated && <Tag color="green">Curated</Tag>}
```

Comment:
Removing this tags will cause the user to don't know what's the status of a definition in the list without inspecting it.
I think it could be useful for a user to know the status at this stage, but i could be wrong.

My past suggestion was not intended to remove these tags, but instead of take the curation status from the /curations API, which require use to make a new /curations call for each component in the list, we could receive those params as additional in the /definitions API, as already made for the urls property.

What do you think?

Reply:
I see. I don't think we should add a status like pending curation to the definition response. This is too time dependent, and we should be able to cache the definition regardless if a PR is open or closed against it.

We could however check the definitions described/tools to see if there are any curations that are part of it. This would allow us to add the 'Curated' tag here. I'll push that update now